http/EffectiveUrlCache.cc

```diff
@@ -168,12 +140,12 @@ shared_ptr<EffectiveUrl> EffectiveUrlCache::get_effective_url(shared_ptr<url> so
         // the instance we placed in the cache - it can be modified and the one in the cache
         // is unchanged. Trusted state was established from source_url when effective_url was
         // created in curl::retrieve_effective_url()
-        effective_url = shared_ptr<EffectiveUrl>(new EffectiveUrl(effective_url));
+        effective_url = make_shared<EffectiveUrl>(effective_url); // shared_ptr<EffectiveUrl>(new EffectiveUrl(effective_url));
```

Comment:
It's a minor point, but commented out code is going to be flagged by sonarscan as a code smell. And it probably improves the code if we remove it.

Reply:
I know, I did this... my bad.

Reply:
Done